aspnetcore/security/authentication/identity.md

```diff
@@ -148,7 +148,7 @@ For more information on scaffolding Identity, see [Scaffold identity into a Razo
 
 ### Examine Register
 
-When a user clicks the **Register** button on the `Register` page, the `RegisterModel.OnPostAsync` action is invoked. The user is created by [CreateAsync](/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.createasync#Microsoft_AspNetCore_Identity_UserManager_1_CreateAsync__0_System_String_) on the `_userManager` object:
+When a user clicks the **Register** button on the `Register` page, the `RegisterModel.OnPostAsync` action is invoked. The user is created by [CreateAsync](/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.createasync) on the `_userManager` object:
```

Comment:
```suggestion
When a user clicks the **Register** button on the `Register` page, the `RegisterModel.OnPostAsync` action is invoked. The user is created by [CreateAsync](/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.createasync#microsoft-aspnetcore-identity-usermanager-1-createasync(-0)) on the `_userManager` object:
```

Reply:
Microsoft_AspNetCore_Identity_UserManager_1_CreateAsync__0_System_String_,The method has only one parameter. The only parameter is string. In fact, there is no corresponding method, so it is deleted.

https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.createasync?view=aspnetcore-6.0&branch=main